src/isolation.rs

```diff
@@ -0,0 +1,61 @@
+use std::{collections::HashMap, ffi::OsString, fs, path::PathBuf};
+
+/// HashMap matching a path in the guest OS ([String]) a path in the host OS ([OsString]).
+pub struct UhyveFileMap {
+	files: HashMap<String, OsString>,
+}
+
+impl UhyveFileMap {
+	/// Creates a UhyveFileMap.
+	///
+	/// Using a list of parameters stored in a [Vec<String>], this function creates
+	/// a HashMap that can match a path on the host operating system given a path on
+	/// the guest operating system.
+	///
```

Comment:
This is probably better located as doc comment to `UhyveFileMap`, but I also see some redundancy.

Reply:
https://github.com/hermit-os/uhyve/commit/7eb72316b004e0d6662296a6f9c4364950bd3756